### PR TITLE
Repair ``TeslaGate`` in ``Room``

### PR DESCRIPTION
**Pull request by louis1706**
-Remove the useless CallDelayed
-Remove useless Error
-Repair the TeslaGate in Room

### CODE REVIEW

**Comment by joker-119** [comment]
1. Don't remove my debug logs.
2. The delay is necessary because lockers don't generate immediately, which is why the delay was added in the first place.

File: Exiled.API/Features/Room.cs [hunk 1]
```diff
@@ -8,7 +8,6 @@
 namespace Exiled.API.Features
 {
     using System;
-#pragma warning disable 1584
```
Comment (joker-119) [inline Exiled.API/Features/Room.cs:11]: Don't remove my pragma disables :anger:

Comment (louis1706) [reply Exiled.API/Features/Room.cs:11]: Oops

**Review by BuildBoy12 — APPROVED** [review]

File: Exiled.Events/Handlers/Internal/MapGenerated.cs [hunk 2]
```diff
@@ -50,23 +50,20 @@ internal static class MapGenerated
...
-                GenerateCache();
-                Door.RegisterDoorTypesOnLevelLoad();
-            });
+            GenerateCache();
+            Door.RegisterDoorTypesOnLevelLoad();
         }
 
         private static void GenerateCache()
         {
+            GenerateTeslaGates();
```
Comment (joker-119) [inline Exiled.Events/Handlers/Internal/MapGenerated.cs:59]: Wont this being before GenerateRooms cause TeslaGate.Room to be null?

Comment (louis1706) [reply Exiled.Events/Handlers/Internal/MapGenerated.cs:59]: Yes it's that

**Pull request merged**